code/03_further_clean_demog_data.R

```diff
@@ -0,0 +1,276 @@
+# ---------------------------------------------------------------------------- #
+# Further Clean Demographic Data
+# Author: Jeremy W. Eberle
+# ---------------------------------------------------------------------------- #
+
+# ---------------------------------------------------------------------------- #
+# Notes ----
+# ---------------------------------------------------------------------------- #
+
+# Before running script, restart R (CTRL+SHIFT+F10 on Windows) and set working 
```

Comment:
Are you familiar with the package `here`? It can do some of this for you

Reply:
Seen it but haven't used yet. Might use this in future!